plugins/websocket_server/WebsocketServer.cc

```diff
@@ -744,6 +744,7 @@ void WebsocketServer::OnMessage(int _socketId, const std::string _msg)
       }
     }
 
+    std::cout << allProtos << std::endl;
```

Comment:
This is always getting print. It would be nice to have it when we run the server with the `-v 4` flag. It's useful information when debugging.